CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
       </trans-unit>
       <trans-unit id="You don't seem to have a subscription. 🤨" xml:space="preserve">
-        <source>You don't seem to have a subscription. 🤨</source>
+        <source>You don't seem to have a subscription. 🤨</source><target>Не зхоже що ви є підписником. 🤨</target>
         <note/>
       </trans-unit>
       <trans-unit id="You have used all your free searches. Subscribe for unlimited searches." xml:space="preserve">
-        <source>You have used all your free searches. Subscribe for unlimited searches.</source>
+        <source>You have used all your free searches. Subscribe for unlimited searches.</source><target>Ви використали усі безкоштовні пошуки. Підпишиться для безлімітних пошуків.</target>
         <note/>
       </trans-unit>
       <trans-unit id="You may know them from" xml:space="preserve">
-        <source>You may know them from</source>
+        <source>You may know them from</source><target>Ви можете знайти їх з</target>
         <note>Header shown for people</note>
       </trans-unit>
       <trans-unit id="Your favorite characters finally get married." xml:space="preserve">
-        <source>Your favorite characters finally get married.</source>
+        <source>Your favorite characters finally get married.</source><target>Ваші улюбленні персонажі нарешті одружуються.</target>
         <note>A fake summary shown when demonstrating hiding spoilers</note>
       </trans-unit>
       <trans-unit id="Your purchase is still pending. Please wait for it to be approved." xml:space="preserve">
-        <source>Your purchase is still pending. Please wait for it to be approved.</source>
+        <source>Your purchase is still pending. Please wait for it to be approved.</source><target>Ваш платіж оброблюється. Будь-ласка, зачекайте на підтвердження.</target>
         <note>Shown when a a purchase needs to be approved; likely a child needs a parent to approve it</note>
       </trans-unit>
       <trans-unit id="Your subscription **will not** auto-renew" xml:space="preserve">
-        <source>Your subscription **will not** auto-renew</source>
+        <source>Your subscription **will not** auto-renew</source><target>Вашу передплату **не буде** оновлено автоматично</target>
         <note>Only shown on a semi-hidden debugging screen</note>
       </trans-unit>
       <trans-unit id="Your subscription **will** auto-renew" xml:space="preserve">
-        <source>Your subscription **will** auto-renew</source>
+        <source>Your subscription **will** auto-renew</source><target>Вашу передплату **буде** оновлено автоматично</target>
         <note>Only shown on a semi-hidden debugging screen</note>
       </trans-unit>
       <trans-unit id="`%@` is unimplemented" xml:space="preserve">
-        <source>`%@` is unimplemented</source>
+        <source>`%@` is unimplemented</source><target>`%@` не реалізовано</target>
         <note>Do not translate</note>
```

Comment:
Do not translate

Reply:
https://github.com/cliss/callsheet-localizations/pull/3/commits/92d8298c05e7a90ebba434bb90661a256defdab3